lib/text/ui_text_displayer.js

```diff
@@ -796,21 +796,35 @@ shaka.text.UITextDisplayer = class {
 
     // The position defines the indent of the text container in the
     // direction defined by the writing direction.
-    if (cue.position != null) {
-      if (cue.writingMode == Cue.writingMode.HORIZONTAL_TOP_TO_BOTTOM) {
-        style.paddingLeft = cue.position;
-      } else {
-        style.paddingTop = cue.position;
-      }
-    }
+    // if (cue.position != null) {
```

Comment:
should this be removed?

Reply:
All changes to this file are not related to this PR, they must be moved to another PR.